.env

```diff
@@ -56,8 +68,5 @@ APP_EXCERPT_MAX_LENGTH=255
 
 ALLOWED_IMAGE_MIME_TYPES='["image/jpeg", "image/png"]'
 
-###> symfony/lock ###
-# Choose one of the stores below
-# postgresql+advisory://db_user:db_password@localhost/db_name
-LOCK_DSN=flock
-###< symfony/lock ###
+SITE_SEND_FROM_EMAIL=detskeriaarhus@example.org
```

Comment:
Please follow https://docs.itkdev.dk/docs/how_we_work/Mail.html when setting default from addresses. So that would be itkdev@mkb.aarhus.dk as default. This is to prevent sending mails to random domains and get banned for sending spam... if some one forgets to update the address when installing the site on a server